### PR TITLE
Sudeep

### PR DESCRIPTION
**Pull request by SudeepMi**

### CODE REVIEW

File: src/TodoCard.js [hunk 1]
```diff
@@ -20,13 +20,13 @@ export default function TodoCard({
   setCompletedItems,
 }) {
 
-  const[disable, setDisable] = useState(0);
+  const[disable, setDisable] = useState(false);
```
Comment (sangya2001) [inline src/TodoCard.js:23]: 0 maynot work as a false but !1 will definitely work.

File: src/CompletedTask.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default function CompletedTask({ completedItems, setCompletedItems }) {
         </div>
         <div className="completedTask-count">{completedItems.length}</div>
       </div>
-
+    { console.log(completedItems)}
```
Comment (sangya2001) [inline src/CompletedTask.js:21]: console should be removed after testing.

File: src/CompletedCard.js [hunk 3]
```diff
@@ -46,9 +46,10 @@ export default function CompletedCard({ description, date, completedItems, setCo
               checkedIcon={<Favorite />}
               onChange={() => heartItem()}
               name="checkedH"
+              checked={ isFav ? true : false }
             />
             <Checkbox
-              disabled={disable}
+              disabled={ isFav ? true : false }
```
Comment (sangya2001) [inline src/CompletedCard.js:52]: rather using ternery use only isFav

**Review by sangya2001 — COMMENTED** [review]
I've added some comments on the code.

File: src/TodoCard.js (hunk 1 above)
Comment (SudeepMi) [reply src/TodoCard.js:23]: ` const[disable, setDisable] = useState(false);`

using boolean instead of 1 & 0.

File: src/CompletedTask.js (hunk 2 above)
Comment (SudeepMi) [reply src/CompletedTask.js:21]: console is removed in the new commit

**Pull request merged**